crates/pathfinder/src/rpc/types.rs

```diff
@@ -470,12 +470,38 @@ pub mod reply {
         }
     }
 
+    // FIXME
+    // throw this conversion away once sync starts passing `sequencer::reply::StateUpdate` instead of
+    // `ethereum::state_update::StateUpdate`
```

Comment:
Cannot suggest in comment-by-comment but it's more greppable if you keep these in one line, or the common email subject/body style if you need more lines.

```
// FIXME: throw this conversion away once ...
```

Reply:
These FIXMEs are not present in the final state.

Reply:
Aah missed that with the commit by commit :)